Products/ZenModel/MetricMixin.py

```diff
@@ -139,10 +139,19 @@ def snmpIgnore(self):
         return False
 
     def getRRDTemplates(self):
-        default = self.getRRDTemplateByName(self.getRRDTemplateName())
-        if not default:
-            return []
-        return [default]
+        templates = []
+        defaultName = self.getRRDTemplateName()
+        replacement = self.getRRDTemplateByName('{}-replacement'.format(defaultName))
+        if replacement:
+            templates.append(replacement)
+        else:
+            default = self.getRRDTemplateByName(defaultName)
```

Comment:
Why do we get **default** only if the replacement is none? Shouldn't the approach be the same as for IpInterface.py?